apps/prerender/src/main.ts

```diff
@@ -12,7 +12,19 @@ const distFolder = path.join(process.cwd(), 'commudle-admin');
 
 app.use(cookieParser());
 const blacklistedUserAgents = ['AhrefsBot', 'AhrefsSiteAudit'];
-prerender.crawlerUserAgents = prerender.crawlerUserAgents.filter((ua) => !blacklistedUserAgents.includes(ua));
+const whitelistedUserAgents = [
+  'GPTBot',
+  'Claude-User',
+  'Claude-SearchBot',
+  'ClaudeBot',
+  'anthropic-ai',
+  'Claude-Web',
+  'Google‑Extended',
```

Comment:
This dash char is a Unicode non-breaking hyphen. Use the standard hyphen instead of this